test/e2e/scrapeconfig_test.go

```diff
@@ -1065,3 +1068,78 @@ var EC2SDTestCases = []scrapeCRDTestCase{
 		expectedError: true,
 	},
 }
+
+var staticConfigTestCases = []scrapeCRDTestCase{
+	{
+		name: "Valid targets",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{
+				{
+					Targets: []monitoringv1alpha1.Target{"1.1.1.1:9090", "0.0.0.0:9090"},
+				},
+			},
+		},
+		expectedError: false,
+	},
+	{
+		name: "Invalid absent targets",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{
+				{},
+			},
+		},
+		expectedError: true,
+	},
+	{
+		name: "Invalid duplicate targets",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{
+				{
+					Targets: []monitoringv1alpha1.Target{"1.1.1.1:9090", "1.1.1.1:9090"},
+				},
+			},
+		},
+		expectedError: true,
+	},
+	{
+		name: "Invalid empty targets",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{
+				{
+					Targets: []monitoringv1alpha1.Target{},
+				},
+			},
+		},
+		expectedError: true,
+	},
+	{
+		name: "Valid labels",
+		scrapeConfigSpec: monitoringv1alpha1.ScrapeConfigSpec{
+
+			StaticConfigs: []monitoringv1alpha1.StaticConfig{
+				{
+					Targets: []monitoringv1alpha1.Target{"1.1.1.1:9090", "0.0.0.0:9090"},
+					Labels:  map[monitoringv1.LabelName]string{"owned-by": "prometheus"},
+				},
+			},
+		},
+		expectedError: false,
+	},
+	{
```

Comment:
this test can be removed then.

Reply:
Right, so lets remove this test from here and create a new PR for the change (or could we do it here, AFAIK it only seems to be in `StaticConfig`).